eclair-node/src/test/scala/fr/acinq/eclair/api/ApiServiceSpec.scala

```diff
@@ -1096,7 +1096,8 @@ class ApiServiceSpec extends AnyFunSuite with ScalatestRouteTest with IdiomaticM
       check {
         assert(handled)
         assert(status == OK)
-        eclair.audit(TimestampSecond.min, TimestampSecond.max)(any[Timeout]).wasCalled(once)
+        val now = TimestampSecond.now()
+        eclair.audit(argThat[TimestampSecond](t => now - 25.hours <= t && t <= now), argThat[TimestampSecond](t => t <= now))(any[Timeout]).wasCalled(once)
```

Comment:
Nit: why use `hours`?
```suggestion
        eclair.audit(argThat[TimestampSecond](t => now - 1.day <= t && t <= now), argThat[TimestampSecond](t => t <= now))(any[Timeout]).wasCalled(once)
```